__tests__/app.test.js

```diff
@@ -136,4 +137,55 @@ describe('GET /api/articles', () => {
         })
     })
     
+})
+describe('GET /api/article/:article_id/comments',() => {
+    it('should return an array of comments for the given article_id', () => {
+     return request(app)
+     .get('/api/articles/1/comments')
+     .expect(200)
+     .then((response) => {
+        expect(response.body.comments.length).toBeGreaterThan(0)
+        const comments = response.body.comments;
+        comments.forEach((comment) => {
+            expect(comment).toHaveProperty('comment_id')
+            expect(comment.comment_id).toBeNumber()
+            expect(comment).toHaveProperty('votes')
+            expect(comment.votes).toBeNumber()
+            expect(comment).toHaveProperty('created_at')
+            expect(comment.created_at).toBeString()
+            expect(comment).toHaveProperty('author')
+            expect(comment.author).toBeString()
+            expect(comment).toHaveProperty('body')
+            expect(comment.body).toBeString()
+            expect(comment).toHaveProperty('article_id')
+            expect(comment.article_id).toBeNumber()
+        })
+     })   
+    })
+    it('GET: 200 should return an array of comments in descending order',() => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.comments.length).toBeGreaterThan(0)
+            expect(response.body.comments).toBeSortedBy('created_at',
+                {descending: true})
+        })
```

Comment:
We could write a test for when an article has no comments associated with it, like for article 2 :)